charts/gateway-operator/ci/node-selector-values.yaml

```diff
@@ -0,0 +1,16 @@
+nodeSelector:
+  karpenter.sh/nodepool: my-nodepool
```

Comment:
This won't work on our CI.

We run these values.yaml files against an actual kind cluster and this label is not set on its nodes. You can try using something generic that we should always see on nodes like `kubernetes.io/os: linux`.

Reply:
thanks for the heads up, just committed the requested changes